app-frontend/src/app/components/sceneDetail/sceneDetail.html

```diff
@@ -0,0 +1,13 @@
+<div class="content">
+  <img ng-if="$ctrl.scene.thumbnails[0]"
+       ng-attr-src="{{$ctrl.scene.thumbnails[0].url}}"
+       class="rounded-img detail-img">
+  <img ng-if="!$ctrl.scene.thumbnails.length"
+       src="http://placehold.it/275x275"
+       class="rounded-img detail-img">
+  <h5 class="color-dark">Image metadata</h5>
+  <dl class="meta-list">
+    <dt ng-repeat-start="(metaKey, metaVal) in $ctrl.scene.sceneMetadata">{{metaKey}}</dt>
```

Comment:
1. I would say for readability purposes sentence case is ideal.
2. I don't think truncating is the answer. I've already been thinking of a slight redesign to this section which should make long metadata more manageable. However if it does become an issue we can be start thinking about truncating. For now let me try and work it out with an alternate layout.